src/main/java/technology/tabula/CohenSutherlandClipping.java

```diff
@@ -74,25 +74,25 @@ public boolean clip(Line2D.Float line) {
         double lineSlope = lineIsVertical ? 0d : (point2.y-point1.y)/(point2.x-point1.x);
 
         while (point1.region != INSIDE || point2.region != INSIDE) {
-            if ((point1.region & point2.region) != INSIDE) return false;
+            if ((point1.region & point2.region) != 0) return false;
```

Comment:
Here (and in other altered Ifs) are comparing the result of the logical OR with zero.